db/controllers/comments.controller.js

```diff
@@ -8,3 +9,24 @@ exports.getAllCommentsByReviewId = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.postNewComment = (request, response, next) => {
+  const { username, body } = request.body;
+  const reviewId = request.params.review_id;
+  if (!username || !body) {
+    return response
+      .status(400)
+      .send({ message: "Bad Request - Missing Required Fields" });
+  } else if (isNaN(reviewId)) {
```

Comment:
postgresql will throw an error if the reviewId is not a number - I have a feeling you have some error handling middleware that might already be handling this.